crates/fluvio-spu-schema/src/server/stream_fetch.rs

```diff
@@ -54,7 +54,7 @@ where
     R: Encoder + Decoder + Default + Debug,
 {
     pub topic: String,
-    pub partition: i32,
+    pub partition: u32,
```

Comment:
Can we use `PartitionId` here?

Reply:
I considered that but `fluvio-spu-schema` does not import `fluvio-types`.
Please tell me if you agree to add the dependency to `Cargo.toml`.

Reply:
I don't see an issue with doing that. @sehz What do you think?

Reply:
should not be problem